addon/components/ilios-calendar-event-month.js

```diff
@@ -11,13 +11,26 @@ export default CalendarEvent.extend({
   event: null,
   timeFormat: 'h:mma',
   classNameBindings: [':event', ':event-pos', ':ilios-calendar-event', 'event.eventClass', ':month-event'],
-  tooltipContent: computed('event', function(){
-    let str = this.get('event.location') + '<br />' +
-      moment(this.get('event.startDate')).format(this.get('timeFormat')) + ' - ' +
-      moment(this.get('event.endDate')).format(this.get('timeFormat')) + '<br />' +
-      this.get('event.name');
-  
-    return str;
+  tooltipContent: computed('event', function() {
+    const location = this.get('event.location');
+    const name = this.get('event.name');
+    const startTime = moment(this.get('event.startDate')).format(this.get('timeFormat'));
+    const endTime = moment(this.get('event.endDate')).format(this.get('timeFormat'));
+    const dueThisDay = this.get('dueThisDay');
+
+    let isILM = this.get('event.name').substring(0, 4) === 'ILM:';
```

Comment:
@jrjohnson `ilmSession` returns a number in two cases:
- event name starts with `ILM: ...`
- event name includes ILM references such as `Required ILM ...`

Returns `undefined` otherwise. See below for three examples. 

<img width="430" alt="screen shot 2015-10-22 at 8 41 23 pm" src="https://cloud.githubusercontent.com/assets/7553764/10684055/97bdf020-78fd-11e5-9969-a24050891b81.png">

Does the event name need to start with `ILM: ...` in order to be considered as an ILM or does the second scenario counts as an ILM?